src/core/shape/vertex.js

```diff
@@ -2253,6 +2253,77 @@ p5.prototype.normal = function(x, y, z) {
   return this;
 };
 
+/** Sets the shader's vertex attribute variables.
+ * 
+ * Shader programs run on the computer's graphics processing unit (GPU)
+ * They live in a part of the computer's memory that's completely separate from
+ * the sketch that runs them. Attributes are variables attached to vertices
+ * within a shader program. They provide a way to attach data to vertices
+ * and pass values from a sketch running on the CPU to a shader program.
+ * 
+ * The first parameter, `attributeName`, is a string with the attribute's name.
```

Comment:
Maybe worth a second sentence saying that it should match the name of the attribute in your shader, e.g. if you've got `attribute vec2 aOffset` (or `in vec2 aOffset` -- as an aside, do you think we should start using glsl es 300 more in the docs?), then you'd `setAttribute('aOffset', yourValue)`

Reply:
I will add this

Reply:
And yes, I definitely think we should move towards es 300 in the docs! I could mention one or both in this reference, and eventually update everything to `in`/`out` in documentation.